packages/fx-core/src/plugins/solution/fx-solution/debug/util/telemetry.ts

```diff
@@ -22,17 +23,17 @@ enum TelemetryPropertyValue {
 }
 
 export enum TelemetryEventName {
-  scaffoldLocalDebugSettings = "scaffoldLocalDebugSettings",
-  setupLocalDebugSettings = "setupLocalDebugSettings",
-  configLocalDebugSettings = "configLocalDebugSettings",
+  scaffoldLocalDebugSettings = "scaffold-local-debug-settings",
+  setupLocalDebugSettings = "setup-local-debug-settings",
+  configLocalDebugSettings = "config-local-debug-settings",
 }
 
 export class TelemetryUtils {
-  static ctx: v2.Context;
+  static telemetryReporter: TelemetryReporter;
```

Comment:
`TelemetryReporter | undefined`?

Reply:
Thanks, forget to change v1 optional reporter and provider to v2 required reporter.